src/main/java/mate/academy/hw02/ImmutableInt.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.hw02;
+
+public class ImmutableInt {
+    private final int value;
```

Comment:
Add ability to change one of the fields, so the class stays immutable

Reply:
I schanged https://github.com/MateAcademy/lessons/pull/15/commits/53b96b74baa7788d478db63a5e5f302331d839bb